File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/system/GetORCAEnvInfo.java
package nta.med.data.model.ihis.system;

public class GetORCAEnvInfo {
	private String orcaIp;
	private String orcaPort;
	private String orcaUser;
	private String orcaPwd;

	public GetORCAEnvInfo(String orcaIp, String orcaPort, String orcaUser, String orcaPwd) {
		super();
		this.orcaIp = orcaIp;
		this.orcaPort = orcaPort;
		this.orcaUser = orcaUser;
		this.orcaPwd = orcaPwd;
	}

	public String getOrcaIp() {
		return orcaIp;
	}

	public void setOrcaIp(String orcaIp) {
		this.orcaIp = orcaIp;
	}

	public String getOrcaPort() {
		return orcaPort;
	}

	public void setOrcaPort(String orcaPort) {
		this.orcaPort = orcaPort;
	}

	public String getOrcaUser() {
		return orcaUser;
	}

	public void setOrcaUser(String orcaUser) {
		this.orcaUser = orcaUser;
	}

	public String getOrcaPwd() {
		return orcaPwd;
	}

	public void setOrcaPwd(String orcaPwd) {
		this.orcaPwd = orcaPwd;
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/System/GetORCAEnvInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.System
{
    [Serializable]
    public class GetORCAEnvInfo
    {
        private String _orcaIp;
        private String _orcaPort;
        private String _orcaUser;
        private String _orcaPwd;

        public String OrcaIp
        {
            get { return this._orcaIp; }
            set { this._orcaIp = value; }
        }

        public String OrcaPort
        {
            get { return this._orcaPort; }
            set { this._orcaPort = value; }
        }

        public String OrcaUser
        {
            get { return this._orcaUser; }
            set { this._orcaUser = value; }
        }

        public String OrcaPwd
        {
            get { return this._orcaPwd; }
            set { this._orcaPwd = value; }
        }

        public GetORCAEnvInfo() { }

        public GetORCAEnvInfo(String orcaIp, String orcaPort, String orcaUser, String orcaPwd)
        {
            this._orcaIp = orcaIp;
            this._orcaPort = orcaPort;
            this._orcaUser = orcaUser;
            this._orcaPwd = orcaPwd;
        }

    }
}